### ISSUE TITLE
Link shortener not working

### ISSUE THREAD

**Issue by YannisDC** [hardillb/node-red-node-physical-web#1]
Somehow I get the message that this link ( http://goo.gl/4gT0xx ) is still too long and should only contain 18 bytes. I thought the http:// was one byte so it would fit. Might this be because the Chrome app is going through a proxy and using my end url? Or am I doing something else wrong?

**Comment by hardillb** [comment]
No, the size checker is a very simple implementation and doesn't do the 1 byte substitution at the moment. I've recently written some some java that did the right things so I'll try and port it over at the weekend

**Comment by YannisDC** [comment]
Okay, I figured. That sir, would be awesome!
